.github/workflows/notify.yaml

```diff
@@ -16,6 +16,9 @@ on:
         description: Notify Boards/Cards
         default: false
 
+  schedule:
```

Comment:
Schedule is OK, but how do we select the values here?

Which values shall be passed to script?

Reply:
SDK Message posting is not called during scheduled run, for Boards/Cards Message we don't need any parameters.